package.json

```diff
@@ -19,6 +19,7 @@
   "devDependencies": {
     "@cypress/react": "^5.10.3",
     "@cypress/vite-dev-server": "^2.2.1",
+    "@itwin/itwinui-css": "^0.51.0",
```

Comment:
why was this added? there's duplicate versions in lockfile now
![](https://user-images.githubusercontent.com/9084735/176438611-15ace9a5-ab16-4b6d-aee8-b716f64a57a9.png)

Reply:
the one which comes from @itwin/itwinui-react has issues with hsl style imports, at least on my local machine, it does even build

Reply:
can you post more info? any error logs?

the only thing i can think of is that sass colors are being used with an older version of sass. there was a similar issue someone else found: https://github.com/iTwin/iTwinUI/discussions/628

Reply:
can you post error logs and the exact place where it fails?

Reply:
created a new pull to remove itwin-css duplicate, must be the issue with node I was using as I needed to jump versions, so it did not build, but I had the same hsl issue that it was not possible to build with that version of itwin-css